bpf/kmesh/cgroup_sock.c

```diff
@@ -40,29 +40,79 @@ struct {
 
 static const char kmesh_module_name[] = "kmesh_defer";
 
-static inline void record_netns_cookie(struct bpf_sock_addr *ctx)
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH);
+    __type(key, __u64);
+    __type(value, __u32);
+    __uint(max_entries, MAP_SIZE_OF_BYPASS);
+    __uint(map_flags, 0);
+} map_of_bypass SEC(".maps");
```

Comment:
can we use the map_of_manager, when bypass set, we remove the key for the particular pod

Reply:
good idea~ `map_of_manager` table can be reused, which reduces the memory required by Kmesh. However, it is not recommended that delete record when bypass set. Otherwise, whether the Pod is bypassed or not managed cannot be determined.
FYI, we can add the value semantics of the records in the `map_of_manager` table.
- If the Pod record exists, the pod is managed by Kmesh.
- The first byte of the value indicates whether to be bypassed.
- The remaining part of the value can be used for scenario extension in the future.

Reply:
I have reused map_of_manager .
- If the Pod record exists, the pod is managed by Kmesh.
- 1: Pod has been bypassed 
- 0: default value, pod is not been bypassed